awscli/examples/schemas/wait/code-binding-exists.rst

```diff
@@ -0,0 +1,13 @@
+**To wait for a new code binding to be created from the put-code-binding API**
+
+The following ``update-registry`` example waits for a new code binding to be created from the put-code-binding API. ::
+
+    aws schemas wait code-binding-exists \
+        --registry-name example-registry \
+        --schema-name example-schema \
+        --schema-version 6 \
+        --language Python36
```

Comment:
There's a mismatch between the command in the code block and that in the description.